front/src/applications/operationalStudies/views/Scenario/components/SimulationResults/SimulationResultsMap.tsx

```diff
@@ -53,9 +53,11 @@ const SimulationResultMap = ({
   // Compute path items coordinates in order to place them on the map
   useEffect(() => {
     const getPathItemsCoordinates = async (path: PathfindingResultSuccess) => {
```

Comment:
A little cleaner with that
```suggestion
    const getPathItemsCoordinates = async ({ path }: PathfindingResultSuccess) => {
```